src/lane_detection/UNet-LaneDetection/Lane_Projection.py

```diff
@@ -0,0 +1,61 @@
+import cv2
+import numpy as np
+
+def project_2D_to_3D_camera(x_array, y_array, depth_to_y_map, f_x, f_y, o_x, o_y):
```

Comment:
Nit: if you could add type hints to these, that would be amazing! tbh idk if this is rly necessary if you just call it all in a specific pipeline and don't expect anyone else to use it.